gateway-ha/src/main/java/io/trino/gateway/ha/security/LbFormAuthManager.java

```
@@ -48,7 +50,11 @@ public LbFormAuthManager(FormAuthConfiguration configuration,
             Map<String, String> pagePermissions)
     {
         this.presetUsers = presetUsers;
-        this.pagePermissions = pagePermissions;
+        this.pagePermissions = pagePermissions.entrySet().stream()
+                .filter(entry -> entry.getKey() != null && entry.getValue() != null)
```

When do we receive null keys?

---

Please throw an exception if it doesn't happen in a real situation.

---

Sure, i added try-catch NPE when doing UpperCase for key.

---

No need to add try-catch block in my opinion. I would just throw an exception.

---

Reminder.

---

I pushed changes instead.

---

Missed it. removed try-catch